skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
I grabbed this from bx-python, which has an [MIT license](https://bitbucket.org/james_taylor/bx-python/src/da37e3aa45dc610ab4fc2f60949de90f93f84401/LICENSE?at=default&fileviewer=file-view-default).

I was under the impression that the MIT license would be compatible with BSD.  @jxtx, are there any issues merging your IntervalTree into scikit-bio?